screens/PointsHistoryScreen.js

```diff
@@ -0,0 +1,44 @@
+import * as Font from 'expo-font';
+import React from 'react';
+import { AsyncStorage, ScrollView, Text, View } from 'react-native';
+import getCustomerTransactions from './historyHelpers';
+
+import Transactions from '../components/Transactions';
+
+export default class PointsHistoryScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      transactions: []
+    };
+  }
+
+  async componentDidMount() {
+    const userId = await AsyncStorage.getItem('userId');
+    getCustomerTransactions(userId).then(transactions => {
+      this.setState({ transactions });
```

Comment:
This was the fix -- when Promises resolve/return some value (for a .then -> .then case), you have to use the result in a .then call. Here we actually want to set the `transactions` object in state so that when we render using `this.state.transactions`, we'll actually have something in that object~